src/axom/core/utilities/FileUtilities.hpp

```diff
@@ -83,6 +83,12 @@ int makeDirsForPath(const std::string& path);
  */
 void getDirName(std::string& dir, const std::string& path);
 
+/*!
+ * \brief Remove the specified file.
+ * \param filename The name of the file.
+ */
+void removeFile(const std::string& filename);
```

Comment:
👍 

We should use this elsewhere in axom, e.g. @bmhan12 recently added code to remove a file in #1073

Reply:
I see he used "int std::remove(const char *)", which I did not know about. The function I added was just so I did not call unistd.h's unlink directly, since I know it's _unlink() on Windows. Would you rather keep removeFile then?

Reply:
I'm ok with either of these for the implementation. I like having the `removeFile()` function in axom's `FileUtilities` since it goes well with the other functions in that file.

Reply:
I made removeFile return a status code (from unlink) and noted it in the doxygen comments. I also made a quick pass and replaced std::remove(filename) with removeFile(filename).